routes/api/request/update-request.js

```diff
@@ -7,5 +7,23 @@ var router = express.Router();
 
 console.log('update-request');
 
+
+router.put("/api/requests", function(req, res) {
+    // Add sequelize code to find a single request where the id is equal to req.params.id,
+    db.Requests.update({
+      detail: req.body.volunteer_id,
+      start_date: req.body.start_date,
+      end_date: req.body.end_date
```

Comment:
`detail: req.body.volunteer_id` should be `detail: req.body.detail`

Also, I'm realizing with the recent additions we've made to request that we should update the following as well:

`address`, `hide_address`, `city`, `state`, `zipcode`

Reply:
updated